.azure-pipelines/ci-build-production.yml

```diff
@@ -0,0 +1,89 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
+
+pr: none
+
+
+stages:
+- stage: "Test"
+  jobs:
+  - job:
+    pool:
+      vmImage: windows-latest
+    strategy:
+      matrix:
+        Node 12:
+          NODE_VERSION: '12.x'
+        Node 14:
+          NODE_VERSION: '14.x'
+        Node 16:
+          NODE_VERSION: '16.x'
+      maxParallel: 2 
+    steps:
+    - template: ./common-templates/security-pre-checks.yml
+    - task: NodeTool@0
+      inputs:
+        versionSpec: $(NODE_VERSION)
+      displayName: 'Install Node.js'
+
+    - script: npm ci
+    - script: npm run build
+    - script: npm run test
+    - script: npm run karma
+      displayName: 'Test the branch'
```

Comment:
Can we please have `displayName` set for each of these script steps?

Reply:
Done